README.md

```diff
@@ -6,8 +6,9 @@
 
 This repository contains IceFlow, a stream processing library based on
 Named-Data Networking (NDN) written in C++.
-Besides the actual library, a number of applications are included in the
-repository illustrating the use of IceFlow for a video processing use case.
+Besides the actual library, a simple wordcount example application is included
```

Comment:
```suggestion
Besides the actual library, a simple word counting example application is included
```
As we're describing the functionality of the example, we should probably not just use the application name here.